action.yml

```diff
@@ -31,4 +31,4 @@ runs:
         args=""
         [ '${{ inputs.pre-release }}' == true ] && args="$args -p"
         [ '${{ inputs.tag }}' == true ] && args="$args -t"
-        gem-push-action -k "${{inputs.key}}" $args ${{inputs.gem-glob}}
+        gem-push-action.sh -k "${{inputs.key}}" $args ${{inputs.gem-glob}}
```

Comment:
I don't like this rename to `.sh`, why is that there? Is it needed for windows support? We have a shebang and the file is executable.

Reply:
shellcheck looks for `script.sh` files; it's configurable but a lot of hassle since the ruby script also has no extension. Easiest move is to add the extension and let the action find it.